hcfb-be/backend-server/app/controller/distribution.controller.js

```diff
@@ -0,0 +1,75 @@
+const db = require('../config/db.config.js');
+const Distribution = db.distributions;
+
+exports.create = (req, res) => {
+    //Save to postgre database
+    Distribution.create({
+      "deliveryLocation" : req.body.deliveryLocation,
+      "deliveryDepartureTime" : req.body.deliveryDepartureTime,
+      "deliveryWeights" : req.body.deliveryWeights,
+      "projectedNumberOfClients" : req.body.projectedNumberOfClients
+    }).then(distribution => {
```

Comment:
You should be able to use async/await here to be more readable than calling Promise methods directly:

https://javascript.info/async-await

So this handler would become:

```js
exports.create = async (req, res) => {
  try {
    const distribution = await Distribution.create({
      deliveryLocation: req.body.deliveryLocation,
      deliveryDepartureTime: req.body.deliveryWeights,
      projectedNumberOfClients: req.body.projectedNumberOfClients,
    });
    res.json(distribution);
  } catch (err) {
    console.log(err);
    res.status(500).json({msg: "error", details: err});
  }
}
```

Right now it's pretty readable either way, but async/await will help keep things that way as we increase complexity.

Reply:
That makes plenty of sense! Does this mean that I should refactor all controller code to utilize async and await keywords?

Reply:
"should" is a strong word, haha. If you have the time and energy for it... it would also be totally fine to merge this as-is and refactor them in later PRs.